ui/src/app/pages/data/data-set/genomic-extraction-modal.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import {Button} from 'app/components/buttons';
+import {ErrorMessage} from 'app/components/messages';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import {dataSetApi} from 'app/services/swagger-fetch-clients';
+
+import {DataSet} from 'generated/fetch';
+
+const {useState} = React;
+
+
+interface Props {
+  dataSet: DataSet;
+  workspaceNamespace: string;
+  workspaceFirecloudName: string;
+  closeFunction: Function;
+}
+
+export const GenomicExtractionModal = ({
+    dataSet, workspaceNamespace, workspaceFirecloudName, closeFunction}: Props) => {
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(false);
+  return <Modal loading={loading}>
+    <ModalTitle>Launch VCF extraction</ModalTitle>
+    <ModalBody>
+      Genomic data extraction will run in background and
+      you’ll be notified when files are ready for analysis.
+      VCF extraction will incur cloud compute costs.
+    </ModalBody>
+    {error &&
+     <ErrorMessage iconSize={16} iconPosition={'top'}>
```

Comment:
nit: the icon positioning feels a little off. not necessary to fix but it could be a nice touch up

Reply:
Good call, fixed (and made `iconPosition` actually optional, default is center which we wanted).